app-devel/mise/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=mise
```

Comment:
I see the upstream provided some shell completions file. https://github.com/jdx/mise/tree/main/completions
Could you install them in a `beyond` file?

Reply:
They also provided man pages: https://github.com/jdx/mise/tree/main/man/man1
Also install them to `/usr/share/man/man1/` pls